### ISSUE TITLE
Fonts not showing correctly on dropdowns

### ISSUE THREAD

**Issue by zBestData** [angular-ui/ui-grid#4285]
Hi,  
Our company is currently looking at Angular and I have been playing with the ui-grid.  I have duplicated the tutorial at: http://ui-grid.info/docs/#/tutorial/101_intro 

The fonts and arrows appear correctly when loading from http://Ui-grid.info, but not when it is run locally.  Am I missing something or is this a bug?(screenshot attached)

Thanks!
Graham

![sample](https://cloud.githubusercontent.com/assets/14015883/9548007/e501b7da-4d6b-11e5-9c5f-6fc840868da9.png)

**Comment by PaulL1** [comment]
http://ui-grid.info/docs/#/tutorial/116_fonts_and_installation

**Comment by zBestData** [comment]
I found it in the ui-grid.css file.  It was looking for the font files in the root directory, not the "fonts" directory as per specs.  I have modified the css file to match and it now works.  I am using Visual Studio and received the package via NuGet.   The modifications I made are below:  

@font-face {
  font-family: 'ui-grid';
  src: url('/fonts/ui-grid.eot');
  src: url('/fonts/ui-grid.eot#iefix') format('embedded-opentype'), url('/fonts/ui-grid.woff') format('woff'), url('/fonts/ui-grid.ttf') format('truetype'), url('/fonts/ui-grid.svg?#ui-grid') format('svg');
  font-weight: normal;
  font-style: normal;
}
